extensions/azurePublish-new/src/components/AzureProvisionDialog.tsx

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+import React from 'react';
+
+import { ChooseProvisionAction } from './ChooseProvisionAction';
+
+export const AzureProvisionDialog = () => {
+  return (
+    <div style={{ height: 'calc(100vh - 65px)' }}>
```

Comment:
Nit: We should pull static Fluent styles out to consts so we don't create a new object every render.